test/ui-test/testSuites/suite_status/shared/steps/chatSteps.py

```diff
@@ -99,9 +103,9 @@ def step(context, emoji_short_name):
 def step(context, emoji_short_name, message):
     _statusChat.send_emoji(emoji_short_name, message)
 
-@Then("the emoji |any| is displayed in the last message")
-def step(context, emoji):
-    _statusChat.verify_last_message_sent(emoji)
+@Then("The last message displayed is \"|any|\"")
```

Comment:
This is the same as `the message |any| is displayed in the last message` on line 114